src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+
+        Figure randomFigure = null;
+
+        int randomType = random.nextInt(5 - 1 + 1) + 1;
+        switch (randomType) {
+            case 1 -> randomFigure = new Circle();
+            case 2 -> randomFigure = new Rectangle();
+            case 3 -> randomFigure = new Square();
+            case 4 -> randomFigure = new IsoscelesTrapezoid();
+            case 5 -> randomFigure = new RightTriangle();
+            default -> getDefaultFigure();
+        }
+        randomFigure.generateRandomParameters();
+        return randomFigure;
+    }
+
+    public Figure getDefaultFigure() {
+        Circle circle = new Circle();
+        circle.color = "WHITE";
+        circle.setRadius(10);
+        circle.name = "Circle";
```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/63095184/d1fd3b86-fd9f-406a-b486-25abb4400606)
Strings too, color you can take from enum

Reply:
not fixed